src/charmed_kubeflow_chisme/testing/README.md

```diff
@@ -4,7 +4,7 @@
 
 ## `deploy_and_assert_grafana_agent`
 
-Helper function to deploy [grafana-agent-k8s](https://charmhub.io/grafana-agent-k8s) to the test model and add cos relations to the charm being tested. This function also checks if the grafana-agent-k8s has reached the desired state, which is blocked with a message composed of two of the following phrases "send-remote-write: off", "grafana-cloud-config: off" or "grafana-dashboards-provider: off".
+Helper function to deploy [grafana-agent-k8s](https://charmhub.io/grafana-agent-k8s) to the test model and add cos relations to the charm being tested. This function also checks if the grafana-agent-k8s has reached the desired blocked state.
```

Comment:
I am thinking about this more and more and perhaps we may not even want to check the blocked status. The thing is, it is not always guaranteed that there will be 0 COS consumers (grafana, loki, prometheus, etc.), so the agent may to into active, but we'll always fail because of this check.

Reply:
I see what you mean, then I guess we can leave this.